src/screens/DiscoveryScreenV2/index.js

```diff
@@ -161,6 +230,9 @@ const DiscoveryScreenV2 = ({route}) => {
           setFollowedDomains={setDiscoveryDataFollowedDomains}
           setUnfollowedDomains={setDiscoveryDataUnfollowedDomains}
           setSearchText={setSearchText}
+          fetchData={fetchDiscoveryData}
+          searchText={searchText}
+          withoutRecent={route.name === 'Followings'}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fetchData` prop is added to the components. Make sure this prop is properly handled in the child components.



<!-- This is an auto-generated comment by OSS CodeRabbit -->